OWML.ModLoader/Owo.cs

```diff
@@ -51,28 +52,36 @@ public void LoadMods()
             var priorityMods = _modFinder.GetMods().Where(mod => mod.Manifest.PriorityLoad).ToList();
             var sortedPriority = _sorter.SortMods(priorityMods);
 
-            var modNames = _modFinder.GetMods().Where(mod => mod.Config.Enabled).Select(mod => mod.Manifest.UniqueName).ToList();
+            var modNames = _modFinder.GetMods().Where(mod => mod.Config.Enabled)
+                .Select(mod => mod.Manifest.UniqueName).ToList();
             var sortedMods = sortedPriority.Concat(sortedNormal);
 
-            foreach (var mod in sortedMods)
+            foreach (var modData in sortedMods)
             {
-                foreach (var dependency in mod.Manifest.Dependencies)
+                var isMissingDependency = false;
+                if (modData.Config.Enabled)
                 {
-                    if (!modNames.Contains(dependency) && mod.Config.Enabled)
+                    var missingDependencies = modData.Manifest.Dependencies
+                        .Where(dependency => !modNames.Contains(dependency));
+                    foreach (var dependency in missingDependencies)
                     {
-                        _console.WriteLine($"Error! {mod.Manifest.UniqueName} needs {dependency}, but it's disabled!");
+                        _console.WriteLine($"Error! {modData.Manifest.UniqueName} needs {dependency}, " +
+                                           "but it's disabled/missing!");
```

Comment:
hm. not too sure on this one. it reduces line length, but it puts a + in a place when you wouldn't expect it

Reply:
not sure what's your problem with it

Reply:
You'd rather have it long?

Reply:
well normally +s are used to concat strings with variables, but this is meant to be one string. i'm pretty sure you can split a string into multiple lines without concat-ing, but it looks ugly 🤷‍♂️
i personally don't mind long lines, but ive proven myself to be the less experienced person here 😄

Reply:
You're right, multiline strings exist in C#, and can be combined with string interpolation. Pushed.